### PR TITLE
global: fixed invenio-search and deps upgrade

### PR DESCRIPTION
**Pull request by ntarocco**
Travis is failing because of the problem with Chrome on Travis, not because of the code changes.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -84,7 +84,7 @@ before_install:
 
 install:
   - "travis_retry pip install -r .travis-${REQUIREMENTS}-requirements.txt"
-  - "travis_retry pip install -e .[all]"
+  - "travis_retry pip install .[all,elasticsearch2]"
```
Comment (slint) [inline .travis.yml:90]: I think dropping the `-e` is ok, unless done differently in other repos.

**Review by slint — APPROVED** [review]

**Comment by ntarocco** [comment]
@slint and @lnielsen I am trying to fix Travis. I will push multiple times.

File: .travis.yml (hunk 1 above)
Comment (egabancho) [reply .travis.yml:90]: I think we put it everywhere, but I don't see why we would need to edit the sources in the travis build 😉

**Comment by ntarocco** [comment]
@lnielsen tests are passing, now `app-ils` should be up-to-date and building if you want to merge before the sprint...
It includes the new `pytest-invenio`.

**Pull request merged**